tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_migrated_valid_invalid.py

```diff
@@ -0,0 +1,288 @@
+            id="EOF1V3540_0004",
+        ),
+        pytest.param(
+            # Type section size incomplete
+            bytes.fromhex("ef00010100"),
+            EOFException.INCOMPLETE_SECTION_SIZE,
+            id="EOF1I3540_0011",
+        ),
+        pytest.param(
+            # Empty code section with non-empty data section
+            bytes.fromhex("ef000101000402000100000400020000000000aabb"),
+            EOFException.ZERO_SECTION_SIZE,
+            id="EOF1I3540_0012",
+        ),
+        pytest.param(
+            # No data section after code section size
+            bytes.fromhex("ef00010100040200010001"),
+            EOFException.MISSING_HEADERS_TERMINATOR,
+            id="EOF1I3540_0017",
+        ),
+        pytest.param(
+            # No data size
+            bytes.fromhex("ef0001010004020001000104"),
+            EOFException.MISSING_HEADERS_TERMINATOR,
+            id="EOF1I3540_0018",
+        ),
+        pytest.param(
+            # Data size incomplete
+            bytes.fromhex("ef000101000402000100010400"),
+            EOFException.INCOMPLETE_SECTION_SIZE,
+            id="EOF1I3540_0019",
+        ),
+        pytest.param(
+            # No section terminator after data section size
+            bytes.fromhex("ef00010100040200010001040002"),
+            EOFException.MISSING_HEADERS_TERMINATOR,
+            id="EOF1I3540_0020",
+        ),
+        pytest.param(
+            # No type section contents
+            bytes.fromhex("ef0001010004020001000104000200"),
+            EOFException.INVALID_SECTION_BODIES_SIZE,
+            id="EOF1I3540_0021",
+        ),
+        pytest.param(
+            # Type section contents (no outputs and max stack)
+            bytes.fromhex("ef000101000402000100010400020000"),
+            EOFException.INVALID_SECTION_BODIES_SIZE,
+            id="EOF1I3540_0022",
+        ),
+        pytest.param(
+            # Type section contents (no max stack)
+            bytes.fromhex("ef00010100040200010001040002000000"),
+            EOFException.INVALID_SECTION_BODIES_SIZE,
+            id="EOF1I3540_0023",
+        ),
+        pytest.param(
+            # Type section contents (max stack incomplete)
+            bytes.fromhex("ef0001010004020001000104000200000000"),
+            EOFException.INVALID_SECTION_BODIES_SIZE,
+            id="EOF1I3540_0024",
+        ),
+        pytest.param(
+            # No code section contents
+            bytes.fromhex("ef000101000402000100010400020000000000"),
+            EOFException.INVALID_SECTION_BODIES_SIZE,
+            id="EOF1I3540_0025",
+        ),
+        pytest.param(
+            # Code section contents incomplete
+            bytes.fromhex("ef0001010004020001002904000000000000027f"),
+            EOFException.INVALID_SECTION_BODIES_SIZE,
+            id="EOF1I3540_0026",
+        ),
+        pytest.param(
+            # Trailing bytes after code section
+            bytes.fromhex("ef000101000402000100010400000000000000feaabbcc"),
```

Comment:
```suggestion
            "ef00 01 01 0004 02 0001 0001 04 0000 00 0080 0000 fe aabbcc",
```
It's currently failing with `EOFException.INVALID_FIRST_SECTION_TYPE`.